src/ClusterManager/test/test_regular_job.py

```diff
@@ -965,6 +968,45 @@ def test_set_max_time_works(args):
 
         state = job.block_until_state_not_in({"running"}, timeout=30)
         assert state == "killed"
+        expected = "running exceed pre-defined 1s"
+
+        details = utils.get_job_detail(args.rest, args.email, job.jid)
+        message = details.get("errorMsg")
+        assert message == expected, "unexpected message " + message
+
+
+@utils.case(dangerous=True)
+def test_set_max_time_in_vc(args):
+    job_spec = utils.gen_default_job_description("regular", args.email,
+                                                 args.uid, args.vc)
+
+    with utils.vc_setting(args.rest, args.vc, args.email,
+                          {"job_max_time_second": 5}):
+        with utils.run_job(args.rest, job_spec) as job:
+            state = job.block_until_state_not_in(
+                {"unapproved", "queued", "scheduling"})
+            assert state == "running"
+
+            state = job.block_until_state_not_in({"running"}, timeout=30)
+            assert state == "killed"
+            expected = "running exceed pre-defined 5s"
+
+            details = utils.get_job_detail(args.rest, args.email, job.jid)
+            message = details.get("errorMsg")
+            assert message == expected, "unexpected message " + message
+
+
+#@utils.case(dangerous=True) should have a way to create a normal user
```

Comment:
We can create a user `dummy` in DB and use it for testing a regular user.

Reply:
I'm thinking create a brand new database before doing tests and tear down after it. We might need a framework to create service and let them discover each other on k8s for testing. Disable this test case for now.